project/src/test/java/com/google/sps/DocumentTest.java

```diff
@@ -0,0 +1,51 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.models;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+import java.util.List;
+import java.util.ArrayList;
+
+@RunWith(JUnit4.class)
+public final class DocumentTest {
+  @Test
+  public void testGetters() {
```

Comment:
Any particular reason there are 2 docs, one with users and one without? If there are no meaningful difference, can just combine everything in doc1. If you expect some behavioral difference, can split into 2 tests: documentWithUserHasAttribute and documentWithoutUserHasAttribute. Generally, unit tests should be modular, and try to test one thing at a time.

Reply:
That is because we were getting a nullptr from datastore when the arraylist was empty, but we figured that out now, so I guess there is no reason for it, I'll just delete it